### PR TITLE
fix: inputs and build

### LINKED ISSUE #32: Fix default for eventWhitelist and Decode
When left blank, the configuration complains about missing `eventWhitelist`. When giving values, it seems to decode them improperly, cf run : https://github.com/ubiquity-os-marketplace/daemon-disqualifier/actions/runs/11316394599/job/31468631289#step:5:93

### PR DESCRIPTION
**Pull request by gentlementlegen**
Resolves #32 
QA: https://github.com/Meniole/user-activity-watcher/actions/runs/11320647090

### CODE REVIEW

File: .github/workflows/update-configuration.yml [hunk 1]
```diff
@@ -1,59 +1,17 @@
...
-            git commit -m "chore: updated generated configuration" || echo "Lint-staged check failed"
-            git push origin HEAD:${{ github.ref_name }}
-          else
-            echo "No changes to commit"
-          fi
+      - uses: ubiquity-os/action-deploy-plugin@main
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          app-id: ${{ secrets.APP_ID }}
+          app-private-key: ${{ secrets.APP_PRIVATE_KEY }}
```
Comment (0x4007) [inline .github/workflows/update-configuration.yml:17]: Kebab case syntax? Is that right?

File: action.yml [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+  eventName:
+    description: "Event Name"
+  eventPayload:
+    description: "Event Payload"
+  settings:
+    description: "Settings"
+  authToken:
+    description: "Auth Token"
+  ref:
+    description: "Ref"
```
Comment (0x4007) [inline action.yml:15]: This could be more specific

File: .github/workflows/update-configuration.yml (hunk 1 above)
Comment (gentlementlegen) [reply .github/workflows/update-configuration.yml:17]: It's usually prettier to use within Actions, but can be changed to camelCase if you want.

Comment (0x4007) [reply .github/workflows/update-configuration.yml]: Yeah I would consider these secret names to be standardized to `APP_ID` and `APP_PRIVATE_KEY` at this point

File: dist/package.json
Comment (0x4007) [inline dist/package.json:1]: This seems unusual

**Review by 0x4007 — APPROVED** [review]
Code looks fine.

File: src/run.ts [hunk 3]
```diff
@@ -12,8 +12,9 @@ export async function run(inputs: PluginInputs) {
     payload: inputs.eventPayload,
     config: inputs.settings,
     octokit,
-    logger: new Logs("verbose"),
+    logger: new Logs("debug"),
```
[the remark this answers is not in the document]
Comment (gentlementlegen) [reply src/run.ts:15]: Yes although the SDK integrates the logger as well so might now be worth changing there right now, I will try to switch this to the SDK later as well.

File: dist/package.json
Comment (gentlementlegen) [reply dist/package.json:1]: When using `type: module` ncc includes this as an ESM reminder from what I've seen.
https://github.com/vercel/ncc?tab=readme-ov-file#usage-1

**Pull request merged**